toolkit/utils/ogmios-client/src/query_ledger_state.rs

```diff
@@ -6,11 +6,14 @@ use crate::{
 };
 use serde::Deserialize;
 
+/// Trait that defines the methods for querying the mainchain ledger state.
```

Comment:
```suggestion
/// Trait that defines the methods for querying the Cardano ledger state.
```
There was some idea that PC can work with different "main chain" where Cardano is one of them, but Ogmios is Cadano specific.